repository/index.go

```diff
@@ -43,7 +44,7 @@ func (idx *Index) store(t pack.BlobType, id backend.ID, pack *backend.ID, offset
 	}
 }
 
-// Store remembers the id and pack in the index.
+// Store remembers the id and pack in the index; an existing entry will be silently overwritten
```

Comment:
Make this full sentences, e.g. `Store remembers the id and pack in the index. An existing entry will be silently overwritten.`